src/assets/styles/theme.js

```diff
@@ -0,0 +1,16 @@
+export const theme = {
+  colors: {
+    white: '#FFFFFF',
+    black: '#111111',
+    lightGrey: '#FBF9FC',
+    darkGrey: '#606060',
+    darkPurple: '#773795',
+    lightPurple: '#ECEFF7',
+  },
+  fontSize: {
+    xl: '34px',
```

Comment:
Możesz spróbować zacząć eksperymentować z innymi jednostkami niż px. W theme często też dodaje się blok "spacing" dla wszelkich paddingów i marginówyciem rem. Jeżeli to cos nowego to możesz poczytać o różnicach między nimi tutaj: https://elementor.com/help/whats-the-difference-between-px-em-rem-vw-and-vh/